packages/test-commons/bootstrap.mjs

```diff
@@ -2,8 +2,10 @@
  * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
  */
 
+import ReactEighteenAdapter from "@cfaester/enzyme-adapter-react-18";
```

Comment:
https://github.com/cfaester/enzyme-adapter-react-18
> "A **very** unofficial adapter for React 18 for [Enzyme](https://enzymejs.github.io/enzyme/)."

Very unofficial, but it at least gets us moving forward until we can replace our Enzyme tests.